README.md

```diff
@@ -1,2 +1,70 @@
-# Math-Magicians
```

Comment:
- [ ] **Let's try to be descriptive in our documentation as it can be of great benefit to users. Kindly format your README using the Microverse template [here](https://github.com/microverseinc/readme-template)**

- [ ] **In your README, remember to add instructions on how to set up and run the app on the local machine.**

Reply:
This is the old README not the new one that I pushed

Reply:
@Ismailco I don't think so. The old one is named `README-old.md`. This is the main README